server/utils/formUtils.ts

```diff
@@ -32,6 +33,16 @@ export default class FormUtils {
     return validationResult(clonedRequest)
   }
 
+  static readonly reasonForChangeId = 'reason-for-change'
+
+  static get validations(): ValidationChain[] {
```

Comment:
why is this in formutils?  Its specific to amend needs and requirements not a generic validator